pkg/restore/interfaces.go

```diff
@@ -54,23 +54,39 @@ type cesRegistry interface {
 	registry.Registry
 }
 
+type cleanupManager interface {
+	cleanup.Manager
+}
+
+// used for mocks
+
+//nolint:unused
+//goland:noinspection GoUnusedType
 type ecosystemRestoreInterface interface {
 	ecosystem.RestoreInterface
 }
 
+//nolint:unused
+//goland:noinspection GoUnusedType
 type statefulSetInterface interface {
 	appsv1.StatefulSetInterface
 }
 
+//nolint:unused
+//goland:noinspection GoUnusedType
 type serviceInterface interface {
 	v12.ServiceInterface
 }
 
-type cleanupManager interface {
-	cleanup.Manager
+//nolint:unused
+//goland:noinspection GoUnusedType
+type appsV1Interface interface {
+	appsv1.AppsV1Interface
 }
 
-// used for mocks
+type coreV1Interface interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `coreV1Interface` is unused